packages/playground/src/pages/Dropin/Dropin.js

```diff
@@ -1,4 +1,5 @@
-import AdyenCheckout from '../../../src';
+import AdyenCheckout from '@adyen/adyen-web';
+import '@adyen/adyen-web/dist/adyen.css';
```

Comment:
Why do we only load this CSS here and not in the other pages?

Reply:
That's fixed now 👍